.gitignore

```diff
@@ -1,5 +1,9 @@
 # Object files
 *.o
+# firmware targets
+*.bin
+*.elf
+*.hex
```

Comment:
.hex are text files so easier to track in the change set of a PR, so I'm not sure it's needed.